lib/peanut.dart

```diff
@@ -37,7 +37,7 @@ Future<Null> run(
   try {
     var args = ['build', '--output', tempDir.path, targetDir];
 
-    Process process = await Process.start('pub', args);
+    Process process = await Process.start('pub', args, runInShell: true);
```

Comment:
Drive by comment: I think you could also have done `Platform.isWindows ? 'pub.dart' : 'pub'` here instead of the runinshell arg.